Modules/Scripted/DICOM/DICOM.py

```diff
@@ -369,10 +400,34 @@ def setup(self):
     # Add spacer to layout
     self.layout.addStretch(1)
 
+  def setDetailsPopup(self, detailsPopup):
+    if self.detailsPopup is not None:
+      self.detailsPopup.loadingFinished.disconnect(self.onLoadingFinished)
+      self.detailsPopup.closed.disconnect(self.onDetailsPopupClosed)
+
+    self.detailsPopup = detailsPopup
+    self.viewFactory.setWidget(self.detailsPopup)
+    self.detailsPopup.loadingFinished.connect(self.onLoadingFinished)
+    self.detailsPopup.closed.connect(self.onDetailsPopupClosed)
+
   def onOpenDetailsPopup(self):
     if not isinstance(self.detailsPopup, self.getSavedDICOMDetailsWidgetType()):
-      self.detailsPopup = self.getSavedDICOMDetailsWidgetType()()
-    self.detailsPopup.open()
+      self.setDetailsPopup(self.getSavedDICOMDetailsWidgetType()())
+
+    layoutNode = slicer.app.layoutManager().layoutLogic().GetLayoutNode()
+    savedViewArrangement = layoutNode.GetViewArrangement()
+    if savedViewArrangement != self.DICOM_BROWSER_LAYOUT_ID:
+
+      centralWidget = slicer.util.mainWindow().findChild("QWidget", "CentralWidget")
+      dataProbe = slicer.util.mainWindow().findChild("QWidget", "DataProbeCollapsibleWidget")
+
+      self.wasCentralWidgetAutoFillBackground = centralWidget.autoFillBackground
+      self.wasDataProbeVisible = dataProbe.isVisible()
+      self.savedViewArrangement = savedViewArrangement
+
+      centralWidget.setAutoFillBackground(False)
```

Comment:
Why autofill background needs to be disabled?

Reply:
Instead of manipulating the parent widget's AutoFillBackground property we could try to set a background for the DICOM widget: "When a widget has a style sheet with a valid background or a border-image, this property is automatically disabled." (http://doc.qt.io/qt-5/qwidget.html#autoFillBackground-prop)

Reply:
When I work on the ShapePopulationViewer, I didn't bother generalizing the approach  ... I wonder if we should improve the layout manager so that it takes care of managing this ?

```
      "<layout type=\"horizontal\">"
      " <item autoFillBackground=\"false\">"
      "  <dicombrowser></dicombrowser>"
      " </item>"
      "</layout>"
```

Reply:
The widget (dicombrowser) knows if it needs autofillbackground or not, so if we really need to control this manually then the widget could have a get function or property that would tell if it requires background fill.

Do you know why dicombrowser is a special case that needs this autofill property manipulation? What happens if we don't set it? Can we set background color using style sheets instead?

Reply:
You're right, setting autoFillBackground to false doesn't seem to be required in this case.